src/eth1spec/rlp.py

```diff
@@ -96,7 +96,7 @@ def BE(x: Uint) -> Bytes:
 
 
 # list encoding/decoding
-def R_l(x: List[RLP]) -> Bytes:
+def R_l(x: Sequence) -> Bytes:
```

Comment:
Missing `[RLP]` generic argument?

Reply:
The issue here is that `isinstance` doesn't let me use generic arguments: 
https://github.com/quilt/eth1.0-specs/blob/8ad5ed2952156f47fb052edd9f32667aaf1e864e/src/eth1spec/rlp.py#L40

Reply:
How about:

```python
from typing import cast

elif isinstance(x, Sequence):
    R_l(cast(Sequence[RLP], x))
```

Reply:
oh neat!